python/mxnet/gluon/block.py

```diff
@@ -1039,7 +1038,9 @@ def forward(self, x):
     """
     def __init__(self):
         super(HybridBlock, self).__init__()
-        self._v2 = inspect.unwrap(self.hybrid_forward.__func__) is HybridBlock.hybrid_forward
+        assert hasattr(self, "hybrid_forward") is False, (
+            "Starting from MXNet2.0, Gluon2.0 with forward interface will be used instead of "
```

Comment:
Does both MXNet2.0 and Gluon2.0 need to be met at the same time?  Propose:
'forward' instead of 'hybrid_forward' interfaces needs to be used starting from Gluon 2.0. ......